django/contrib/admin/static/admin/js/core.js

```diff
@@ -85,6 +85,12 @@ function findPosY(obj) {
         return (this.getSeconds() < 10) ? '0' + this.getSeconds() : this.getSeconds();
     };
 
+    Date.prototype.getAbbrevMonthName = function() {
+        return typeof window.CalendarNamespace === "undefined"
+            ? this.getTwoDigitMonth()
+            : this.getFullMonthName().substring(0, 3);
```

Comment:
```suggestion
            : window.CalendarNamespace.monthOfYearAbbrev[this.getMonth()];
```

Reply:
I remember I changed this, should have double checked on GitHub :sweat_smile: